modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/FieldsSelectorDropdown.js

```diff
@@ -57,7 +57,7 @@ const FieldsSelectorDropdown = ({fields}) => {
 						'component-action': Liferay.FeatureFlags['LPS-193005'],
 					})}
 					displayType="secondary"
-					size="xs"
+					size={Liferay.FeatureFlags['LPS-193005'] ? 'xs' : undefined}
```

Comment:
@matuzalemsteles This perfectly fixes part of the issue, that was a regression from your PR. First column size is unrelated to your change, I'll take a closer look and try to figure it out.